src/ValueParsers/BoolParser.php

```diff
@@ -9,12 +9,12 @@
  *
  * @since 0.1
  *
- * @license GPL-2.0+
+ * @license GPL-2.0-or-later
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
 class BoolParser extends StringValueParser {
 
-	const FORMAT_NAME = 'bool';
+	private const FORMAT_NAME = 'bool';
```

Comment:
Did you do some check for usage in projects using this library?

Should also be mentioned as a breaking change in the release notes in README.md

Reply:
I'll have a look myself at usages soon

Reply:
Did not spot any usages